test/test_nestedtensor.py

```diff
@@ -8343,14 +8343,6 @@ def __torch_dispatch__(self, func, types, args=..., kwargs=None):
         sample_match_fn=lambda device, sample: ("(T, NT)" in sample.name),
         name="broken_copysign_backward",
     ),
-    # chunk(): backward doesn't work for the batch dim yet
-    XFailRule(
-        error_type=RuntimeError,
-        error_msg="Nested Tensor doesn't support chunk backward on dim=0 yet",
-        op_match_fn=lambda device, op: (op.full_name == "chunk"),
-        sample_match_fn=lambda device, sample: ("batch_dim" in sample.name),
-        name="broken_chunk_backward",
-    ),
```

Comment:
May need to update the `test_copy_` test which asserts `copy_ only supports Nested Tensors that have same size and the exact same offset tensor`